### PR TITLE
instructor homepage: 'are you sure you want to publish' alert can be more informative. #3338

### LINKED ISSUE #3338: instructor homepage: 'are you sure you want to publish' alert can be more informative.
image

e.g. is should be 'publish responses'. We can also mention that students will receive an email.

### PR DESCRIPTION
**Pull request by MrFool**
Fixes #3338

### CODE REVIEW

**Comment by wkurniawan07** [comment]
My 2 cents: try to leave easy issues? Else you're depriving the newer contributors.
Especially this one. Looks like a `forFirstTimers` issue for me.

**Comment by MrFool** [comment]
Sure we can, it says forContributors though, maybe should change it to forFirstTimers instead

**Comment by wkurniawan07** [comment]
Hahaha I mean for issues that are, like, the moment you see it you immediately know how to do it and 5 mins later it's done. Not necessarily labelled. Remember, both of us got this far because there are issues like this for us in the past, don't deprive the newer contributors :smile:

**Comment by MrFool** [comment]
I understand :p so then we can label the sister issue forFirstTimers then, this can serve as a reference since it's done already :x

**Comment by wkurniawan07** [comment]
Another 2 cents from me: try not to refactor (even minor things such as adding/deleting space/empty lines) parts that are not part of the issue okay? I know the feeling of itching to fix such mistakes. I get it almost all the time. But the reason to leave them is much more important.
If there are bugs involving those lines, we can git blame straightaway to the commit that made the lines. Remember issue #3273? That's how I managed to find the guilty commit in no time. Had someone refactored it, it would take me longer. Not impossible, but longer, but let's make other people's lives easier okay?
What's important is we follow the coding standard. Let the mistakes of the past contributors be the mistakes of the past contributors.

File: src/main/webapp/js/instructor.js [hunk 1]
```diff
@@ -72,7 +72,8 @@ function toggleDeleteFeedbackSessionConfirmation(courseID, name) {
  * @param name
  */
 function togglePublishEvaluation(name) {
-    return confirm("Are you sure you want to publish the session " + name + "?");
+    return confirm("Are you sure you want to publish the responses for the session " + name + "?" + 
+            " An email will be sent to students to inform them that responses are ready for viewing.");
```
Comment (LowWeiLin) [inline src/main/webapp/js/instructor.js]: Emails might not be sent, depending on the settings for the session.

Comment (MrFool) [inline src/main/webapp/js/instructor.js]: In that case, should we just not tell them about the email? Referring to #3338

**Comment by MrFool** [comment]
screen shot 2015-05-07 at 1 11 12 pm
screen shot 2015-05-07 at 1 11 06 pm
screen shot 2015-05-07 at 1 10 50 pm
screen shot 2015-05-07 at 1 10 44 pm

**Comment by damithc** [comment]
I'm ok with the alerts. May be put the session name within double quotes?

**Comment by MrFool** [comment]
Sure thing, will get to that

**Comment by MrFool** [comment]
screen shot 2015-05-09 at 1 54 31 am

Updated

**Pull request merged**